Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+      ENV LC_ALL C.UTF-8
+      ENV LANG C.UTF-8
+RUN apt-get update && apt-get install -y \
+    python3 python3-pip
+
+RUN mkdir /oauth-proxy
+COPY . /oauth-proxy/
+WORKDIR /oauth-proxy
+RUN cd /oauth-proxy && pip3 install -r requirements.txt
+RUN python3 /oauth-proxy/create_keys.py
```

Comment:
Should this better be run in entrypoint?

Reply:
yes will move it to entry point thanks